console/console-init/ui/src/modules/project/ProjectPage.tsx

```diff
@@ -149,6 +158,49 @@ export default function ProjectPage() {
       }
     }
   };
+  const handleOnChangeEnable = (projects: IProject[], toEnable: boolean) => {
+    let iotQueryVariables: Array<{ name: string; namespace: string }> = [];
+    selectedProjects.forEach(
+      (project: IProject) =>
+        project.name &&
+        project.namespace &&
+        project.projectType === ProjectTypes.IOT &&
+        iotQueryVariables.push({
+          name: project.name,
+          namespace: project.namespace
+        })
+    );
+    if (iotQueryVariables.length > 0) {
+      const queryVariable = {
+        a: iotQueryVariables,
+        status: toEnable
+      };
+      setToggleIoTProjectQueryVariables(queryVariable);
+    }
+  };
+
+  const onToggleAll = (action: string) => {
+    const toEnable = action === "enable" ? true : false;
```

Comment:
can you try variable name status instead of toEnable.

Reply:
status doesn't sound great for boolean. I used toEnable because it is based on action i.e `toEnable` or `toDisable`